packages/core/src/baseCogniteClient.ts

```diff
@@ -126,29 +99,10 @@ export default class BaseCogniteClient {
       options.retryValidator ?? createUniversalRetryValidator();
     this.http = this.initializeCDFHttpClient(baseUrl, options);
 
-    if (options.authentication) {
-      const { credentials, provider } = options.authentication;
-
-      this.credentialsAuth = new CredentialsAuth(
-        this.httpClient,
-        credentials,
-        provider
-      );
-    }
-
-    if (isBrowser() && !isUsingSSL() && !options.noAuthMode) {
-      console.warn(
-        'You should use SSL (https) when you login with OAuth since CDF only allows redirecting back to an HTTPS site'
-      );
-    }
```

Comment:
Wouldn't this warning be logged even if we're not using legacy auth?

Reply:
Yes, but it was placed there due to legacy auth. No reason to restrict to https anymore.
The https restriction came from legacy auth.